Cargo.toml

```diff
@@ -45,8 +45,9 @@ futures = { version = "0.3.1", optional = true }
 http = { version = "0.2.0", optional = true }
 tokio = { version = "0.2.13", optional = true }
 url = "2.1.1"
-kv-log-macro = "1.0.4"
+kv-log-macro = "1.0.5"
 mime_guess = "2.0.3"
+femme = { path = "../../femme" }
```

Comment:
I'd like to try it out. Can you post which version of `femme` is used?

Reply:
@isgj https://github.com/lrlna/femme/pull/9

Reply:
When I build the app with the release flag the logs look like
```json
{"level":30,"time":1589717147052,"msg":"--> Response sent","method":"\"GET\"","status":"200","duration":"\"157ms\"","path":"\"/api/v1/users\""}
```
the string (of key: value, not the message) have also the double quotes escaped. Is that intended?

Reply:
@isgj that's definitely not expected and we should fix that. Filed https://github.com/lrlna/femme/issues/12 on femme. Thanks for reporting!

Reply:
Okay, this will require patching the `log` crate. I don't think we should block adding this feature on that, even if it might be worth fixing in either femme or tide for the time being.